TRANSFORMERS/ARITHMETIC/utils/arithmetic_utils.py

```diff
@@ -0,0 +1,13 @@
+from flojoy import OrderedPair, Scalar, Vector
+
+
+def get_param_keys(
```

Comment:
I would pick a better name for this function (Rethinking about it, `get_val` wasn't that bad of a name)